Decred Wallet/Info.plist

```diff
@@ -47,6 +47,8 @@
 		<string>SourceSansPro-Bold.ttf</string>
 		<string>SourceSansPro-SemiBold.ttf</string>
 	</array>
+    <key>NSFaceIDUsageDescription</key>
+    <string>Face Authentication</string>
```

Comment:
If this is a message displayed to users when requesting authentication with face id, then change to `Continue with FaceID` or something more user-friendly.

Reply:
its a key for permission

Reply:
its a key for permission